src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -723,4 +723,9 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Property SourceProvider is null.</value>
     <comment>Don't localize 'SourceProvider'</comment>
   </data>
+  <data name="Error_AttemptingToPackSDKproject" xml:space="preserve">
+    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. To force nuget to run `pack` command, you can set the '{0}' environment variable to 'true' but this is not recommended.</value>
```

Comment:
```suggestion
    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. To force NuGet to run `pack` command, you can set the '{0}' environment variable to 'true' but this is not recommended.</value>
```

@JonDouglas also recommended to remove the last phrase...

```suggestion
    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. To force NuGet to run `pack` command, you can set the '{0}' environment variable to 'true'.</value>
```

Reply:
#### Alternate version 1

```suggestion
    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild -t:Pack` to pack this project instead. To force NuGet's `pack` command through, you can set the '{0}' environment variable to 'true' but this is not recommended.</value>
```

#### Reason for the change

The user has already run the Pack command, so the phrasing should be, `To force the command through`. Anyway, this is what I thought. If it's too difficult for non-native users, then there is no need to update that phrase.

Reply:
#### Alternate version 1

```suggestion
    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild -t:Pack` to pack this project instead. You can override this behavior by setting the '{0}' environment variable to 'true'.</value>
```

@JonDouglas recommended [removing the last phrase](#discussion_r986174259).